### PR TITLE
fix: add assets to public folder

### PR DESCRIPTION
**Pull request by LunrEclipse**
Added Rich Link Preview dependent assets such as font and network logos to public folder

### CODE REVIEW

**Review by zzmp — APPROVED** [review]

File: public/images/logos/Arbitrum_Logo.png
Comment (JFrankfurt) [inline public/images/logos/Arbitrum_Logo.png:1]: We already have `src\assets\svg\arbitrum_logo.svg`, can that work for you here instead?

Comment (LunrEclipse) [reply public/images/logos/Arbitrum_Logo.png:1]: I have not figured out how to bundle svgs with cloudflare functions so pngs are preferred

Comment (LunrEclipse) [reply public/images/logos/Arbitrum_Logo.png:1]: also the cloudflare functions need the files to be bundled with the functions folder or in the public folder to access

Comment (JFrankfurt) [reply public/images/logos/Arbitrum_Logo.png:1]: Can you look into moving those files to the public folder and bundling svgs?

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**